docs/aks/README.md

```diff
@@ -351,65 +322,68 @@ helm repo update
 
 ```
 
-## Install Linkerd Service Mesh into the cluster
+### Install AAD Pod Identity for the application
 
-Download the Linkerd v2 CLI:
+Change directories to the `docs/aks` folder and make the `aad-podid.sh` script executable. Running this shell script will deploy AAD Pod Identity to your cluster and assign a Managed Identity.
+
+>NOTE: The second command below has a `.` then a space followed by `./aad-podid.sh ...` this is so the exported variables in the script persist after the script ends in the uder interactive shell
 
 ```bash
 
-# macOS and linux/WSL
+export MI_Name=${He_Name}-mi
 
-curl -sL https://run.linkerd.io/install | sh
-export PATH=$PATH:$HOME/.linkerd2/bin
+cd $REPO_ROOT/docs/aks
+sudo chmod +x aad-podid.sh
+
+. ./aad-podid.sh -a ${He_AKS_Name} -r ${He_App_RG} -m ${MI_Name}
```

Comment:
why are we "sourcing" the shell script vs. just running it?  Should it be  ./aad-pod-id.sh ...  ?

Reply:
I'll respond to this cause I ran into the same issue :)
When I ran it directly, the variables in the script were not exported, which are needed for setup later. I do remember seeing a note somewhere in this doc about why the script is run this way.